myosuite/tests/test_tutorials.sh

```diff
@@ -13,7 +13,8 @@ pip3 install scikit-learn
 # Install potential missing packages needed for the tutorials
 pip3 install jupyter ipykernel tabulate matplotlib torch h5py
 pip3 install git+https://github.com/aravindr93/mjrl.git@pvr_beta_1vk # install from branch `pvr_beta_1vk` compatible with `mujoco` native binding
```

Comment:
can we have this fix from Arvind in mjrl main? this here forces to pull from the branch but users installing from main might not find easily this fix